### PR TITLE
Async client.

### PR DESCRIPTION
**Pull request by koriakin**

### CODE REVIEW

File: python/veles/async_client/node.py [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+from veles.proto import messages
+from veles.proto.node import PosFilter
+from veles.schema.nodeid import NodeID
+
+from veles.proto.exceptions import (
+    SchemaError,
+)
+
+
+def _resolve_obj(conn, obj):
```
Comment (altran01) [inline python/veles/async_client/node.py]: We already have identical function in other `node.py` in other module - maybe we should create some module with common utility functions?

File: python/proxy [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import sys
+import signal
+import importlib
+
+from veles.server.conn import AsyncLocalConnection
+from veles.server.proto import create_unix_server, create_tcp_server
+from veles.async_client.conn import AsyncRemoteConnection
+from veles.async_client.proto import create_unix_client, create_tcp_client
+
+print('Świtezianka proxy starting up...')
```
Comment (altran01) [inline python/proxy]: WTF H4X.
Took me a second to understand what do you do here.

File: python/veles/async_client/node.py [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+# limitations under the License.
+
+import asyncio
+
+from veles.async_conn.node import AsyncNode
+from veles.proto import messages
+from veles.proto.node import PosFilter
+from veles.schema.nodeid import NodeID
+
+from veles.proto.exceptions import (
```
Comment (koriakin) [inline python/veles/async_client/node.py]: Done

File: python/veles/async_client/node.py (hunk 1 above)
Comment (koriakin) [inline python/veles/async_client/node.py]: This function has been fixed.

File: python/veles/async_client/conn.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import weakref
+
+from veles.async_conn.conn import AsyncConnection
+
```
Comment (koriakin) [inline python/veles/async_client/conn.py]: Done.

**Pull request merged**